service/src/main/resources/config/aou/test/SC2023Q3R1/sql/t_person.sql

```diff
@@ -1,4 +1,41 @@
-SELECT
-    p.person_id, p.year_of_birth, p.birth_datetime, p.gender_concept_id, p.race_concept_id, p.ethnicity_concept_id
-FROM `all-of-us-ehr-dev.SC2022Q4R6.person` p
-
+SELECT p.person_id,
+       p.year_of_birth,
+       p.birth_datetime,
+       p.gender_concept_id,
+       p.race_concept_id,
+       p.ethnicity_concept_id,
+       p.sex_at_birth_concept_id,
+       CASE
+           WHEN asum.person_id IS NULL THEN 0 ELSE 1 END has_fitbit_activity_summary,
+       CASE
+           WHEN hrml.person_id IS NULL THEN 0 ELSE 1 END has_fitbit_heart_rate_level,
+       CASE
+           WHEN hrs.person_id IS NULL THEN 0 ELSE 1 END has_fitbit_heart_rate_summary,
+       CASE
+           WHEN si.person_id IS NULL THEN 0 ELSE 1 END has_fitbit_steps_intraday,
+       CASE
+           WHEN sds.person_id IS NULL THEN 0 ELSE 1 END has_fitbit_sleep_daily_summary,
+       CASE
+           WHEN sl.person_id IS NULL THEN 0 ELSE 1 END has_fitbit_sleep_level,
+       CASE
+           WHEN sl.person_id IS NULL AND sl.person_id IS NULL AND hrs.person_id IS NULL
+            AND si.person_id IS NULL AND sds.person_id IS NULL THEN 0 ELSE 1 END has_fitbit
+FROM `all-of-us-ehr-dev.SC2023Q3R1.person` p
+         LEFT JOIN
+     (SELECT DISTINCT person_id
+      FROM `all-of-us-ehr-dev.SC2023Q3R1.activity_summary`) asum ON (p.person_id = asum.person_id)
```

Comment:
Not for this PR, but I hope we can remove these flags in the future. Will follow-up offline.

Reply:
That would be good.  More importantly, need to support direct values (boolean is missing, String is missing)/ some SQL like `distinct(column_name) from t_person` for attributeMapping in t_person
```
      "has_fitbit_xxx": {
        "value": { "column": "has_bitbit_xx" },
        "display": { ["true","false"])
or
        "display": {"rawSql??": "SELECT DISTINCT has_fitbit_xx FROM t_person""}
 
```